### PR TITLE
[release-v1.24] Auto pick #1689: Upgrade ECK image to 1.8.0

### PR DESCRIPTION
**Pull request by Brian-McM**
Cherry pick of #1689 on release-v1.24.

#1689: Upgrade ECK image to 1.8.0

### CODE REVIEW

**Review by doublek — APPROVED** [review]
LGTM

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

**Pull request merged**
